salt/orch/update.sls

```diff
@@ -282,3 +269,17 @@ services_setup:
       - dex
     - require:
       - cni_setup
+
+{% if masters|length > 0 %}
+masters-remove-update-grain:
+  salt.function:
+    - tgt: L@{{ masters.keys()|join(',') }}
```

Comment:
Can't we just use?

```
- tgt: G@roles:kube-master and G@update_in_progress:true
- tgt_type: compound
```

Reply:
We can, it was just to make sure to use the exact same subset that was used in the rest of the orchestration. Do you want me to change it or keep it as is?